format/spec.md

```diff
@@ -1060,6 +1076,14 @@ The types below are not currently valid for bucketing, and so are not hashed. Ho
 | **`float`**        | `hashLong(doubleToLongBits(double(v))` [4]| `1.0F` ￫ `-142385009`, `0.0F` ￫ `1669671676`, `-0.0F` ￫ `1669671676` |
 | **`double`**       | `hashLong(doubleToLongBits(v))`        [4]| `1.0D` ￫ `-142385009`, `0.0D` ￫ `1669671676`, `-0.0D` ￫ `1669671676` |
 
+For multi-arg hash function, the hash value is the result of `hashBytes` on the concatenation of the bytes of the arguments. NULL input is ignored.
+
+| Struct examples                    | Hash Specification                                                                    | Test value                                                 |
+|------------------------------------|---------------------------------------------------------------------------------------|------------------------------------------------------------|
```

Comment:
I feel like if a table is an appendix table, it should be complete, (cover all types).  What do you think about something like?

For multiple arguments, hashBytes() is applied on the concatenated byte representation of each argument:
| primitive type |  byte representation |
 | int | littleEndianBytes(long(v)) |
| ... | ... |

For example, the hash representation of (a:int, b:string) will be hashBytes(concatenation(littleEndianBytes(long(v)), utf8Bytes(b))